packages/blocks/src/page-block/edgeless/edgeless-page-block.ts

```diff
@@ -90,6 +90,17 @@ import {
 import { xywhArrayToObject } from './utils/convert.js';
 import { getCursorMode, isCanvasElement, isFrameBlock } from './utils/query.js';
 
+export interface EdgelessViewport {
```

Comment:
We already have a `PageViewport` interface, and I think `EdgelessPageBlockComponent` and `DocPageBlockComponent` can share the same interface.

https://github.com/toeverything/blocksuite/blob/ee3a4ff482d2459a58cdfa2bb752d055e3774f05/packages/blocks/src/page-block/doc/doc-page-block.ts#L24-L33

Reply:
@Flrande Done